src/autorest-core/lib/pipeline/schema-validation.ts

```diff
@@ -23,12 +23,14 @@ export function GetPlugin_SchemaValidator(): PipelinePlugin {
     const errors = await new Promise<{ code: string, params: string[], message: string, path: string }[] | null>(res => validator.validate(obj, extendedSwaggerSchema, (err, valid) => res(valid ? null : err)));
     if (errors !== null) {
       for (const error of errors) {
+        const errorString = JSON.stringify(error, null, 2);
```

Comment:
this might fail if the error is not serializable as json(Contains circular reference)

you could use `util.inspect` from node which lets you specify a max depth